src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -384,23 +378,47 @@ private void populateOmopTable() throws Exception {
   public void testSnapshotBuilder() throws Exception {
     populateOmopTable();
 
-    // Test getConcepts
-    var getConceptResponse = dataRepoFixtures.getConcepts(steward, datasetId, 2);
-    var concepts = getConceptResponse.getResult();
+    var concept1 =
+        new SnapshotBuilderConcept().name("concept1").id(1).count(22).code("11").hasChildren(false);
+    var concept3 =
+        new SnapshotBuilderConcept().name("concept3").id(3).count(24).code("13").hasChildren(true);
+
+    getConceptChildrenTest(concept1, concept3);
+    searchConceptTest(concept1);
+    getConceptHierarchyTest(concept1, concept3);
```

Comment:
I like breaking the test cases down like this.
(not for this PR) As an alternate approach, would it make sense to break the snapshot builder into its own integration test and use the `@BeforeOnce` to share the resources set up in populateOmopTable()?

Reply:
That's a good point, I don't want the overhead of ingesting the table multiple times and it could be shared across tests (I think the junit5 annotation is `@BeforeAll`). But I wonder if it relies on a dataset that's deleted between tests? I can create a ticket to look into this.

Reply:
I created https://broadworkbench.atlassian.net/browse/DC-1032 to look into this possibility.